images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -8,6 +8,21 @@ function Get-OSVersion {
     return "OS Version: $OSVersion Build $OSBuild"
 }
 
+function Get-AvailableDiskSpace {
+    $drives = @("C", "D")
```

Comment:
I thought that D drive doesn't exist during image-generation and it is attached only during deployment? But I could be wrong

Reply:
Yep, I've forgot about it. I don't think it's a good idea to hardcode expectable mount size, so seems like, we should remove info about it completely.

Reply:
Fixed.